tests/unit/suites/mfx_dispatch/linux/mfx_dispatch_test_cases_libs.cpp

```diff
@@ -84,15 +85,24 @@ TEST_P(DispatcherLibsTestParametrized, ShouldEnumerateCorrectLibNames)
 #elif defined(__x86_64__)
     const std::string LIBMFXSW("libmfxsw64.so.1");
     const std::string LIBMFXHW("libmfxhw64.so.1");
+    const std::string ONEVPLRT("libmfx-gen.so.1.2");
 #endif
 
     std::string modules_dir(MFX_MODULES_DIR);
     mfxIMPL impl = GetParam();
     std::vector<std::string> libs;
 
-    if (MFX_IMPL_BASETYPE(impl) == MFX_IMPL_AUTO ||
-        MFX_IMPL_BASETYPE(impl) == MFX_IMPL_AUTO_ANY)
-    {
+    eMFXHWType platform = MFX_HW_UNKNOWN;
+    auto devices = get_devices();
+    if (devices.size()) {
+        platform = devices[0].platform;
+    }
+
+    if (platform == MFX_HW_UNKNOWN) {
```

Comment:
Add a code comment that we load VPL for all unknown platforms which correspond to future platforms. And make condition upside down, i.e.:
```
if (platform != MFX_HW_UNKNOWN) {
  load.emplace_back(l"ibmfxhw64.so");
} else {
  load.emplace_back(l"ibvpl64.so");
}
```

Reply:
Done